### ISSUE TITLE
Unexpected item included in the final mask

### ISSUE THREAD

**Issue by ohheysherry66** [NathanUA/U-2-Net#50]
Congratulations for your amazing work with u2net！
Recently,I have tried your net on portrait segmentation task.Also I trained it from scratch by portrait segmentation dataset without other objects;
It got great performance.However,I find that sometimes,objects such as chairs and street nameplates can also be included. I am confused about it. 
Since I trained it from scratch by the given dataset.It can be seen as a semantic  segmentation task,right?Why the other object can be inclued?
Thankyou!

**Comment by PytaichukBohdan** [comment]
> Congratulations for your amazing work with u2net！
> Recently,I have tried your net on portrait segmentation task.Also I trained it from scratch by portrait segmentation dataset without other objects;
> It got great performance.However,I find that sometimes,objects such as chairs and street nameplates can also be included. I am confused about it.
> Since I trained it from scratch by the given dataset.It can be seen as a semantic segmentation task,right?Why the other object can be inclued?
> Thankyou!

Could you please share the process of your training?
Did you freeze layers (which ones)? Which parameters & optimizer did you choose? Maybe some other tricks?

Thanks in advance!

**Comment by ohheysherry66** [comment]
> > Congratulations for your amazing work with u2net！
> > Recently,I have tried your net on portrait segmentation task.Also I trained it from scratch by portrait segmentation dataset without other objects;
> > It got great performance.However,I find that sometimes,objects such as chairs and street nameplates can also be included. I am confused about it.
> > Since I trained it from scratch by the given dataset.It can be seen as a semantic segmentation task,right?Why the other object can be inclued?
> > Thankyou!
> 
> Could you please share the process of your training?
> Did you freeze layers (which ones)? Which parameters & optimizer did you choose? Maybe some other tricks?
> 
> Thanks in advance!

Hi,I just use the original optimizer the writer provided in the code.I use the pretrained model and add the other layer to predict boundary by freeze all parameters and random init the added layer,but it seems help a little.
